src/status_im/contexts/wallet/events.cljs

```diff
@@ -8,6 +8,7 @@
     [status-im.contexts.wallet.data-store :as data-store]
     [status-im.contexts.wallet.events.collectibles]
     [status-im.contexts.wallet.item-types :as item-types]
+    [status-im.contexts.wallet.sheets.scanned-address.view :as scanned_address]
```

Comment:
nit: in cljs should use `-` in favor of `_`

Reply:
i.e `scanned-address`

Reply:
Done